debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/J9IndexableObjectHelper.java

```diff
@@ -196,7 +303,7 @@ public static boolean isInlineContiguousArraylet(J9IndexableObjectPointer objPoi
 	}
 
 	/**
-	 *  @param objPointer array object who's elements we are outputting to dst
+	 *  @param objPointer array object whose elements we are accessing
 	 *  @param dst destination array where we will output the elements
 	 *  @param start starting index of the elements we are interested in
 	 *  @param length number of elements to output 
```

Comment:
While you're fixing this javadoc, please add a summary and remove the extra space before each `@param` tag.